app/views/tradingName/ConfirmRemoveTradingNameView.scala.html

```diff
@@ -15,28 +15,71 @@
  *@
 
 @import config.AppConfig
+@import forms.YesNoForm._
 
 @this(mainTemplate: MainTemplate,
       govukBackLink: GovukBackLink,
-      govukButton: GovukButton)
+      govukRadios: GovukRadios,
+      govukErrorSummary: GovukErrorSummary,
+      govukButton: GovukButton,
+      formWithCSRF: FormWithCSRF)
 
-@(tradingName: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(form: Form[YesNo], tradingName: String)(implicit messages: Messages, appConfig: AppConfig, request: Request[_])
 
-@mainTemplate(pageTitle = messages("confirmTradingNameRemove.title")) {
+@mainTemplate(pageTitle =
+        if (form.errors.nonEmpty) messages("common.error.prefixTitle", messages("confirmRemove.title", tradingName))
+        else messages("confirmRemove.title", tradingName)) {
 
   @govukBackLink(BackLink(
```

Comment:
This back link is incorrect; we would want it to go to the "change-remove" option page.